docs/guides/src/main/server/health.adoc

```diff
@@ -0,0 +1,62 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Enabling Keycloak Health checks"
+summary="Learn how to anble and use Keycloak health checks"
```

Comment:
```suggestion
summary="Learn how to enable and use Keycloak health checks"
```

Reply:
@andreaTP this minor typo is still there, could you commit this suggestion? We can squash when me merge, so no need to do that on your end

Reply:
This is solved:
https://github.com/keycloak/keycloak/pull/10396/files#diff-bc0bf6337cbce2297d204b55a4f483380d91faa3a82bd97febb83f104fcbd9c9R8